.appveyor.yml

```diff
@@ -0,0 +1,59 @@
+environment:
+  ARMADILLO_DOWNLOAD: "http://ftp.fau.de/macports/distfiles/armadillo/armadillo-8.400.0.tar.xz"
+  ARMADILLO_LIBRARY: "C:/projects/mlpack/armadillo-8.400.0/build/Debug/armadillo.lib"
```

Comment:
Is this used? I don't think there's even a C:/projects/mlpack/ folder for ensmallen. It seems to build just fine in any case.

Reply:
I think you're right---nice catch. :+1: I believe that on Windows it will only link against LAPACK and BLAS directly, so `ARMADILLO_LIBRARY` will be ignored.  Let's see if it still builds if I remove it, and if so, I'll go ahead and merge. :+1: